### ISSUE TITLE
Documentation update

### ISSUE THREAD

**Issue by oadams** [persephone-tools/persephone#199]
Things to document:
- [ ] The need to build Kaldi for `filterbank_and_pitch` features.
- [ ] Need for settings.ini / include a default.
- [ ] Model training arguments.
- [ ] Use of train_prefixes.txt, valid_prefixes.txt, test_prefixes.txt
- [ ] Mention Python 3.7 is not supported by Tensorflow.
- [ ] Point users to Github issue tracker to raise problems.
- [ ] Document assumptions about data, and how the tool is primarily aimed at the single-speaker scenario.

**Comment by shuttle1987** [comment]
Is there any test data for `filterbank_and_pitches`? I'm putting together environment build automation over in PR #208 and I'd like to be able to verify that I can automate the install of Kaldi by running some data through these code paths.

**Comment by oadams** [comment]
There's no data: the distinction between `filterbank` and `filterbank_and_pitches` is only that pitch features are extracted from the audio additionally, so a test for pitch extraction could use the same data as any existing test that extracts filterbank features.

**Comment by shuttle1987** [comment]
In that case perhaps we can just use an existing test set for the data but with the different parameters for this? I'd like to get some test that requires Kaldi to work

**Comment by oadams** [comment]
Yes, that's right.

**Comment by shuttle1987** [comment]
Is this work related to #86 and #1 ?

**Comment by oadams** [comment]
Yeah, perhaps we should just enumerate current documentation change requirements in the checklist above, unless its really big enough and requires implementation that somehow warrants its own issue.
